content/docs/start/data-pipelines.md

```diff
@@ -304,23 +306,23 @@ stages:
 DVC pipelines (`dvc.yaml` file, `dvc run`, and `dvc repro` commands) solve a few
 important problems:
 
-- _Automation_ - run a sequence of steps in a "smart" way that makes iterating
+- _Automation_: run a sequence of steps in a "smart" way which makes iterating
   on your project faster. DVC automatically determines which parts of a project
-  need to be run, and it caches "runs" and their results, to avoid unnecessary
-  re-runs.
-- _Reproducibility_ - `dvc.yaml` and `dvc.lock` files describe what data to use
+  need to be run, and it caches "runs" and their results (in the aptly-named
+  <abbr>run-cache</abbr>) to avoid unnecessary reruns.
```

Comment:
Good intent but TMI prob. and complicates the reading. It's an intro doc, no need to specifically mention the run-cache out here (it's OK in the hidden section above tho 👍).

Reply:
```suggestion
  need to be run, and it caches "runs" and their results to avoid unnecessary reruns.
```